sdrgui/mainwindow.cpp

```diff
@@ -3065,7 +3065,7 @@ void MainWindow::keyPressEvent(QKeyEvent* event)
     }
 }
 
-void MainWindow::orientationChanged(Qt::ScreenOrientation orientation) const
+void MainWindow::orientationChanged(Qt::ScreenOrientation orientation)
```

Comment:
Any valid reason to remove const ?

Reply:
Yes, "setTabPosition" ,ethod is non-const, so calling it from const method isn't correct.